src/axom/quest/MarchingCubes.cpp

```diff
@@ -149,7 +153,7 @@ void MarchingCubes::computeIsocontour(double contourVal)
 axom::IndexType MarchingCubes::getContourNodeCount() const
 {
   axom::IndexType contourNodeCount =
-    m_singles.empty() ? 0 : m_facetCount * m_singles[0]->spatialDimension();
+    !m_domainCount ? 0 : m_facetCount * m_singles[0]->spatialDimension();
```

Comment:
This might read a bit clearer if it explicitly tested against 0 since `m_domainCount` is an int not a boolean:
```cpp
  axom::IndexType contourNodeCount = (m_domainCount == 0) 
      ? 0 
      : m_facetCount * m_singles[0]->spatialDimension();
```

Reply:
Done.